t/md5.t

```diff
@@ -1,11 +1,20 @@
 use strict;
 use warnings;
 
-use Test::More tests => 3;
+use Test::More tests => 4;
+use File::Temp qw(tempfile);
 use Rex::Commands::MD5;
 
 my $test_file = Rex::Helper::File::Spec->catfile( 't', 'md5test.bin' );
 
+my ( $fh, $strangely_named_file ) = tempfile(
```

Comment:
First I thought it would be nice to use the same way for both test files (either commit both to the repo, or generate both during tests). But then I wondered if it would be enough to just use the new generated file, so we can get rid of the other one. Hmm, I don't see any obvious downsides to that yet :thinking: 

What do you think? Should we drop the old one, or keep both but handle them the same way (either generated or committed)?

Reply:
Alright, I chose to have a single generated temporary file for MD5 tests.